lms/services/roster.py

```diff
@@ -97,6 +91,78 @@ def fetch_roster(self, lms_course: LMSCourse) -> None:
             update_columns=["active", "updated"],
         )
 
+    def fetch_assignment_roster(self, assignment: Assignment) -> None:
```

Comment:
This method is very similar to the existing one for courses.

I have tried to use just one method an parametrize a few bits but picking the right table and right columns read worse that the current version.